src/graph.jl

```diff
@@ -22,6 +22,31 @@ end
 
 Graph() = Graph(0)
 
+function Graph{T<:Real}(adjmx::SparseMatrixCSC{T})
+    dima, dimb = size(adjmx)
+    if dima != dimb
+        error("Adjacency / distance matrices must be square")
+    elseif !issym(adjmx)
+        error("Adjacency / distance matrices must be symmetric")
```

Comment:
"Undirected Adjacency matrices must be symmetric", because Directed Adjacency matrices are not symmetric.